main.go

```diff
@@ -160,3 +160,36 @@ func main() {
 		log.Fatalf("HTTP shutdown: %v", err) // nolint:gocritic
 	}
 }
+
+func (e *elizaServer) Introduce(
+	ctx context.Context,
+	req *connect.Request[elizav1.IntroduceRequest],
+	stream *connect.ServerStream[elizav1.IntroduceResponse],
+) error {
+	name := req.Msg.Name
+	if name == "" {
+		return connect.NewError(connect.CodeInvalidArgument, fmt.Errorf(
+			"name is required",
+		))
+	}
```

Comment:
I think it may be a little annoying to display errors with the current demo. Maybe we can fall back to "anonymous visitor" if the name is empty?